<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationForm</class>
 <widget class="QWidget" name="StationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="StationGraph" native="true">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>0</y>
     <width>610</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="stationlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>STATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HVONButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>HV ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HVOFFButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ROUGHLINE ON</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>PN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PNLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Serial</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SerialLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>TestDate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TestDateLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>PO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="POLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>RCV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RCVLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Shipped</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ShippedLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>RFID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RFIDLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>98</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Addr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PumpAddrSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>CH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PumpCHSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>181</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Gun off GC closed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="GunnOffGCClosedLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>181</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>License Plate Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LPNLineEdit"/>
    </item>
   </layout>
  </widget>
  <zorder>StationGraph</zorder>
  <zorder>pushButton</zorder>
  <zorder>stationlabel</zorder>
  <zorder>HVONButton</zorder>
  <zorder>HVOFFButton</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>layoutWidget_8</zorder>
  <zorder>layoutWidget_9</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>LPNLineEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>PumpAddrSpinBox</zorder>
  <zorder>label</zorder>
  <zorder>PumpCHSpinBox</zorder>
  <zorder>label_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
